apps/src/main/resources/application.conf

```diff
@@ -0,0 +1,25 @@
+akka {
+  actor {
+    provider = "cluster"
+  }
+
+  remote {
+    log-remote-lifecycle-events = off
```

Comment:
I know we often have this in tests and examples, but I think we should use default to see if the default settings are annoying (and should be changed). Btw, I don't think this is used by Artery.